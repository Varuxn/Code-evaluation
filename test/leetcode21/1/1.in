100 100
shn
koaegcifzdeksbtokg
akqnvoepnpjlplqlm
tmddszmujgzcfpapinwobacxsopxmcisbrvgdsdfrlnr
upovfkanxliyvmqincilzheuqtqlfzpfotpbqukxnolwniecth
ttgjpotsngxlchdrhdikloly
cako
hhylmsgbahivaqecmvucikxblzjvfunnerjfhfzqidpavcpey
zzpmivlktkufiecjruvmjvy
gktupwhghe
l
aqonyebjokdchudpgrctfltwtfoyuyydttsrsllu
k
ihxek
etrqzkukzsobmlk
bufssgslscsijgnztuthisucceeuplfxsdrbiui
hkezluldfscfxjjp
qbllytckjtykbbsoirydmxmaouyorgncvqalgmqdrnqlf
xixnpiavfnsyvmkxoztbyn
zuxaedgplmsbxcqvavoknafajvaakvmmawobm
yotkubrdbkahdirpfcwbfvmbe
btjpkvropo
nkppgwjikaebosclmebmbwc
gfwbuavxikyquftzri
dohudqpappogtdzunhlquulwqaoz
ocncjwenjqjflnagdlxfv
i
fyrduyfupfktu
iqthmbbwhydxfsoqwewr
mmdzoihmdkuyywfw
cenyeivkisrhflorxgpxcuj
sfktoy
blzblbidoqsxlinqarhdqarjmb
x
wstwflztbsue
xbgncjshv
o
zfllbpcxrlshblrticucihknwsybdjhco
nscjhlhcjgxgwjpqpzxcppofeb
dfclceyikoddenjkwnvdny
ybwlsmhtondagimialycnrs
kbcikld
ffoatmpznedwhdtkmjutazuyzmcekbnbjmugfuswzyocqpg
mbfpnbplm
gcdsmyendgkecllppejetjjytyekwaiissbknlfwcoogqmnbru
ikyjtfhrrsnkciggimzftlih
gvrt
gkfhcqepwkopsrvcxu
krdkxawzhovuhnilmgqqsxabanpoutcgp
dahyliqdifktsvpjajzmhmhlbkjvqevuhgobrlhcanfijlzw
qgtfcrcwypvrgvaykeyzwtflzkwkuaqwlqglyedljfcslbrdlk
gyuizlsvjjieznbatxdsklkakeznalnhvdebbowz
fgqzjhzsljqsquvrzrwlvqyblwtzuikdljsxjbgyxadxthw
wfmutvzzthcfgkhxnfxrtldaiujxflzorvilsxnkrcejda
wrdkrhihffijpvgzj
aekgseinjmzaiochucqdkwmuwjbeqefykngfhwjnmreya
gdciyvuefce
gohivamiaejitnetiwm
qyhswfiemlsfnwabhlkdmgrkekqlvdozfttqvs
riynwqjunymguveqcmqggnbfje
sdzwodhbzgmsnfrqopvxbwqwjtozhtmewpjprswucxkxnt
htbturlrrmfwuqjascljajc
tlnqsaqfbyqebbfdmxymyyllzranukuoirgjcqjycmei
dlqotvqyqbxpmwraauharkwizrpwdgiwxtvkz
cnrtkseg
klpxy
dkuhewlcubxacgmitlo
cuuyrgymyllgxhngxehrnldirqldpyev
auimyzueb
qodzk
xyrsybpffsaujodgukcaxngethgrsz
w
fliuixibcdmheofttnmjqywrrfknriaqycpvyyceyviz
yccpbvcttutuiu
xvnxvpwqlbkqdfbebjfgttombexcudacd
khkhuniviabkhclaukdjswxlsaaztso
if
lpsktiassleoh
kypcjcvqathzekqrntkzz
dcqddnk
dekhggpdxhfmmuzvdisykgdqrxgenzuspvfuvqxuul
ojkwoatzofgbihwdneizbmjyofbqfuxkfhwg
fzrhmz
icshclmqonstiubmlmyygduencxsrghvduv
jrvamdqsajmxjyjmxwplkyhdwqtxlivzlebejl
zhphpslyibmdchxkfcprq
izpkgspzkcunynymsvpyzxbvamwmrekudhaqkzror
vpfjqbsygqxvjtmkrxabddr
boifbqxgyzagflsuwgluvunyogmlizunerzdbizuepwuuav
ucm
agisvoauuzmdrjjk
rmrmmrisykywgpyejgimkoupbsiuyu
hholmbcigflnfwiequnkrywyiyi
ybbrevqqayzkwehkgqfiphrplmsafzhi
uheugjyozgaeudmkqvkww
yydcasdchgrfkowaohbsjqfdgfiuvymhhb
ijethdsfvefywxi
ylppzxuvnrdodtjvogenvqvge
ik
bmhuhzxbfhgudaujxdnefe
zfzkatfqetcgvdbcing etrqzkukzsobmlking cakoed oqwerqwe ytjojszoeyzjfksnqjbgfyjling muvgining uoucewdmpvrxftqsbmtmhgacnsuijssclvtrdvuwpoing pxxqbaypmynscybxqbeed yuuoqcpooybdkebiing fyrduyfupfktuing gqweing fjkvoed dcqddnked pjtvgued gkyfodmgeigcjsibnmotdimjrcolz zsvtzetqkaegevccynbufsfbhxwpgfeges aekgseinjmzaiochucqdkwmuwjbeqefykngfhwjnmreyaed dkuhewlcubxacgmitloqwerqwe aekgseinjmzaiochucqdkwmuwjbeqefykngfhwjnmreyaed fzrhmzed xyrsybpffsaujodgukcaxngethgrszing hfsxpzcppydysyqzgrcpuzlkqvcing noing vjnmrfmwwonbmntxbikvxkyckmyamhgvsriufwfhxwing otmgnuing xing tgkjbmmcwkwyxklchqauilgkdykdgnvbfkazptwnshzed tocnlfgqltwfhj gktupwhgheed gohivamiaejitnetiwming bmhuhzxbfhgudaujxdnefeed dcqddnking gktupwhgheed fliuixibcdmheofttnmjqywrrfknriaqycpvyyceyvizes hoihfscgodpjfguyavjuhbihsugbiowctdurlfnaydjhdmdsed kpbafnonyvaktoybdolkxxuyqwnefcles dlqotvqyqbxpmwraauharkwizrpwdgiwxtvkzing ycdtijhsrjiwdrwfiuwyzppzksing yagazcwfixvswulkeed zxipzcxlryrtbvlprmsciynlkujkgufxqtpfgceeocphnoing xoylhvjbqabhqarfwrritgufeaermiwbiqpzwjqj yydcasdchgrfkowaohbsjqfdgfiuvymhhbed elhekhyoczyocjeing ygkmttduclzhking qbllytckjtykbbsoirydmxmaouyorgncvqalgmqdrnqlfes sahes rdiwqjmtkktticflwfeselutxicwhjwring fqetlllhavxeuuvgisxqpgsyktpjexzddtvoeing ojbjfgsgiabwghzjbkjfjyalxzkotsnifbitrzued hbhnxudqing xfiazokfdkzwkjaqwpqfvisvlsjoed jwjzpkfspkgbyniing gyuizlsvjjieznbatxdsklkakeznalnhvdebbowzed srrnunikaloftwmeejkeborrfoanzrflbvokkbypves khkhuniviabkhclaukdjswxlsaaztsoing ojrifffswvhkpewkomujlmeascvxpahfxuosing ting obwvcdklidcqugkgrwajgmixyeyeoxdxbxkkiing axbbanrelvssvycwmging vpfjqbsygqxvjtmkrxabddring agisvoauuzmdrjjking ygkvwbqwerqwe mbsrdyopirted fgqzjhzsljqsquvrzrwlvqyblwtzuikdljsxjbgyxadxthwing hholmbcigflnfwiequnkrywyiyi rmrmmrisykywgpyejgimkoupbsiuyuing pffoycuued shnes inhaxugvusfcuwqucghpgmptocpakqfjthbtgytxrdylulnyqwerqwe zuxaedgplmsbxcqvavoknafajvaakvmmawobmed dohudqpappogtdzunhlquulwqaozing gyuizlsvjjieznbatxdsklkakeznalnhvdebbowzed ifing sytkfed qtvnrqnoagssciavzofrqozzqqcqing uunyorspkykaaukimbvjdmrehwsreuyvfecsgdes koaegcifzdeksbtokged sqkefkrzcbhing dekhggpdxhfmmuzvdisykgdqrxgenzuspvfuvqxuuled kbciklding cltyupjdketadmovudymlwuvwtnhdbrmsqfhfqg khkhuniviabkhclaukdjswxlsaaztsoing uheugjyozgaeudmkqvkww fzrhmzing wstwflztbsueing ffoatmpznedwhdtkmjutazuyzmcekbnbjmugfuswzyocqpging mwmlgyhgkzhvgvtxrclhvfrpfmsixqxing dkuhewlcubxacgmitloes kjhywzwcaed uckcktfkfmqphbgigykbkcarkyxtauvwxsxupymbsqjuudyes yfrswjnrpceglyanhicysthcycmqgausxnspnxwce ybbrevqqayzkwehkgqfiphrplmsafzhied ylppzxuvnrdodtjvogenvqvgeing rbed gvrted fvabgklbdbrnzujwvnwujmykgsbiidthrjqaswtzwing wwhhcnkqxlgwxzilxdrcgixoyuhfcjjing fzqiicoxesrkgdwjlkuszfning shned piwpphuwgzed
