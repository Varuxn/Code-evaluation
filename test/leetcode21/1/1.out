zfzkatfqetcgvdbcing etrqzkukzsobmlk cako o ytjojszoeyzjfksnqjbgfyjling muvgining uoucewdmpvrxftqsbmtmhgacnsuijssclvtrdvuwpoing pxxqbaypmynscybxqbeed yuuoqcpooybdkebiing fyrduyfupfktu gqweing fjkvoed dcqddnk pjtvgued gkyfodmgeigcjsibnmotdimjrcolz zsvtzetqkaegevccynbufsfbhxwpgfeges aekgseinjmzaiochucqdkwmuwjbeqefykngfhwjnmreya dkuhewlcubxacgmitlo aekgseinjmzaiochucqdkwmuwjbeqefykngfhwjnmreya fzrhmz x hfsxpzcppydysyqzgrcpuzlkqvcing noing vjnmrfmwwonbmntxbikvxkyckmyamhgvsriufwfhxwing o x tgkjbmmcwkwyxklchqauilgkdykdgnvbfkazptwnshzed tocnlfgqltwfhj gktupwhghe gohivamiaejitnetiwm bmhuhzxbfhgudaujxdnefe dcqddnk gktupwhghe fliuixibcdmheofttnmjqywrrfknriaqycpvyyceyviz hoihfscgodpjfguyavjuhbihsugbiowctdurlfnaydjhdmdsed k dlqotvqyqbxpmwraauharkwizrpwdgiwxtvkz ycdtijhsrjiwdrwfiuwyzppzksing yagazcwfixvswulkeed zxipzcxlryrtbvlprmsciynlkujkgufxqtpfgceeocphnoing x yydcasdchgrfkowaohbsjqfdgfiuvymhhb elhekhyoczyocjeing ygkmttduclzhking qbllytckjtykbbsoirydmxmaouyorgncvqalgmqdrnqlf sahes rdiwqjmtkktticflwfeselutxicwhjwring fqetlllhavxeuuvgisxqpgsyktpjexzddtvoeing o hbhnxudqing x jwjzpkfspkgbyniing gyuizlsvjjieznbatxdsklkakeznalnhvdebbowz srrnunikaloftwmeejkeborrfoanzrflbvokkbypves k o ting o axbbanrelvssvycwmging vpfjqbsygqxvjtmkrxabddr agisvoauuzmdrjjk ygkvwbqwerqwe mbsrdyopirted fgqzjhzsljqsquvrzrwlvqyblwtzuikdljsxjbgyxadxthw hholmbcigflnfwiequnkrywyiyi rmrmmrisykywgpyejgimkoupbsiuyu pffoycuued shn i zuxaedgplmsbxcqvavoknafajvaakvmmawobm dohudqpappogtdzunhlquulwqaoz gyuizlsvjjieznbatxdsklkakeznalnhvdebbowz i sytkfed qtvnrqnoagssciavzofrqozzqqcqing uunyorspkykaaukimbvjdmrehwsreuyvfecsgdes k sqkefkrzcbhing dekhggpdxhfmmuzvdisykgdqrxgenzuspvfuvqxuul k cltyupjdketadmovudymlwuvwtnhdbrmsqfhfqg k uheugjyozgaeudmkqvkww fzrhmz w ffoatmpznedwhdtkmjutazuyzmcekbnbjmugfuswzyocqpg mwmlgyhgkzhvgvtxrclhvfrpfmsixqxing dkuhewlcubxacgmitlo k uckcktfkfmqphbgigykbkcarkyxtauvwxsxupymbsqjuudyes yfrswjnrpceglyanhicysthcycmqgausxnspnxwce ybbrevqqayzkwehkgqfiphrplmsafzhi ylppzxuvnrdodtjvogenvqvge rbed gvrt fvabgklbdbrnzujwvnwujmykgsbiidthrjqaswtzwing w fzqiicoxesrkgdwjlkuszfning shn piwpphuwgzed